docs/scripts/todo.md

```diff
@@ -0,0 +1,7 @@
+# To do
+* Anchors are in place, but the redirection system needs to be defined. 
+    * What to use as anchor: the full IRI, or just the suffix? This implementation assumes that the document has a common prefix, so that anchors are the IRI suffixes.
+
+* Test github pages deployment
+
+* Set up github workflows
```

Comment:
Issues might be a better way of keeping track on things we want to do than such a file. Anyway, some possible issues to add may be:
* also document relations (EMMOntoPy can serialise to manchester syntax)
* proper handling of versions

Reply:
True, this todo was added for my own purpose but I should get more comfortable using github issues

Reply:
BTW the todo.md file can be safely deleted later